test/functional/terminal/buffer_spec.lua

```diff
@@ -170,6 +176,30 @@ describe(':terminal buffer', function()
       ^                                                  |
       :6put a                                           |
     ]])
+  end
+
+  it('sends data to the terminal when the "put" operator is used', function()
+    test_terminal_put_operator()
+  end)
+
+  it('sends data to the powershell terminal when the "put" operator is used', function()
+    if not helpers.has_powershell() then
+      pending("powershell not found", function() end)
+      return
+    end
```

Comment:
can use skip()

```suggestion

    skip(helpers.has_powershell(), 'missing powershell')
```

Reply:
> merge the powershell tests into the non-powershell tests

Something like this?
```
  it('sends data to the terminal when the "put" operator is used', function()
    test_terminal_put_operator()
    if helpers.has_powershell() then
      helpers.set_shell_powershell(false)
      test_terminal_put_operator()
    end
  end)
```
This^ fails.  Probably needs to clear the terminal window in between the two tests.

Or did you mean something else?

Reply:
yes

Reply:
> Probably needs to clear the terminal window in between the two tests.

I don't know how to do this.  I tried calling `clear()` but that pauses/halts the test.

Reply:
oh I see the `before_each` is doing too much extra stuff. Ok, let's keep the separate tests :)

I pushed a commit